client/src/components/DataFiles/DataFilesListing/DataFilesListingCells.js

```diff
@@ -161,3 +163,22 @@ export const FileIconCell = ({ cell }) => {
 FileIconCell.propTypes = {
   cell: PropTypes.shape({ value: PropTypes.string }).isRequired
 };
+
+export const ViewPathCell = ({ file }) => {
+  const dispatch = useDispatch();
+  const onClick = () => {
+    dispatch({
+      type: 'DATA_FILES_TOGGLE_MODAL',
+      payload: { operation: 'showpath', props: { file } }
+    });
+  };
+  return (
+    <Button color="link" onClick={onClick} styleName="viewpath-root">
+      <h6>View Path</h6>
```

Comment:
__Minor Change__: I appreciate the attempt at consistency. But, please do not make this a heading. It should not be, and I have begun to remove this where I find it. Making this change has the desired visual result of the text **not** being bold.

```suggestion
      View Path
```